### DISCUSSION TITLE
一行代码千般喜 | /cn/2019/04/one-loc-joy/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1372] [Comments]
https://yihui.org/cn/2019/04/one-loc-joy/

[a comment by an automated account is not shown]

**Comment by chuxinyuan** [comment]
我好像也有这样的体会：寒冷的冬天，戴两只手套还是觉得冻手，可是戴一只就能感觉到有一只手是温暖的。

**Comment by chuxinyuan** [comment]
复制第二段，我发现行内代码和文字的高度不一样高，有割裂感，我不知道如何描述这个现象。

**Reply by yihui** [reply]
代码字体我略微缩小了：https://github.com/yihui/yihui.org/blob/655a3846c8b48e9fbe930f3a4b02870d74fe7d1a/static/css/fonts.css#L22-L24

你系统上的等宽字体和我的不一样（我用的是苹果），我不知道你的字体看起来是什么样的。

**Reply by chuxinyuan** [reply]
![](https://xmin.netlify.app/images/manjaro-yihui-site.png)

这是 Manjaro 系统上看到的效果。

**Reply by chuxinyuan** [reply]
我觉得字体缩放 95% 是一个方面，选中后的蓝色背景的高度和行内代码的括号的高度等高，而其它字符的蓝色背景高度是明显高于字的。

**Reply by chuxinyuan** [reply]
> 这是 Manjaro 系统上看到的效果。

对不起，忘了告诉你我用的火狐浏览器，我在 Win10 下用火狐浏览器和 Edge 浏览器又试了下，Edge 浏览器下一切正常。上图的现象似乎只有在火狐浏览器上才能重现。

**Reply by yihui** [reply]
通常来说，小写英文字符本身就会比中文方块字矮、只占三格线的两格，再加上字体缩小了 5%，就更矮了。

浏览器之间的渲染差异是我懒得管的，尤其是这也不是什么大事，不妨碍阅读。

**Reply by chuxinyuan** [reply]
是的，确实没必要纠结于这个可以忽略的形式。我安装了 Chromium 浏览器，确保下次向你报告问题的时候，排除浏览器之间的渲染差异。
